app/helpers/application_helper.rb

```diff
@@ -113,6 +113,12 @@ def fa_icon(icon, attributes = {})
     content_tag(:i, nil, attributes.merge(class: class_names.join(' ')))
   end
 
+  def material_symbol(icon)
+    content_tag(:i, nil, class: 'material-symbol') do
+      inline_svg("@material-symbols/svg-600/outlined/#{icon}.svg")
```

Comment:
Currently pulling these straight from the npm package similar to what the JS modules are doing. Could hypothetically streamline/shorten this with webpack config or other prep .. but also this might be fine.